mix.exs

```diff
@@ -45,7 +45,11 @@ defmodule Appsignal.Plug.MixProject do
 
     [
       {:plug, ">= 1.1.0"},
-      {:appsignal, ">= 2.1.5 and < 3.0.0"},
+      {
+        :appsignal,
+        ">= 2.1.5 and < 3.0.0",
```

Comment:
```suggestion
        ">= 2.2.8 and < 3.0.0",
```

Probably need to change this. See https://github.com/appsignal/appsignal-elixir/pull/713#issuecomment-965345406

Reply:
Done!